kola/tests/ignition/oem.go

```diff
@@ -29,20 +29,50 @@ func init() {
 		Distros:     []string{"cl"},
 		// This test overwrites the grub.cfg which does not work on cloud environments after reboot
 		Platforms:  []string{"qemu", "qemu-unpriv"},
-		MinVersion: semver.Version{Major: 2983},
-		UserData: conf.ContainerLinuxConfig(`storage:
+		MinVersion: semver.Version{Major: 3550},
```

Comment:
I don't know what to think about this version bump - this means that `cl.ignition.oem.regular` won't be tested on currently supported versions in other channels.